### PR TITLE
Spliting dump request to add Query 🐛 🚩 

### PR DESCRIPTION
**Pull request by ancogamer**
Spliting []dump into dumps, and editing the dump[1] to = req.URL.String(), so we can tests itens that depends on querys from the request.

### CODE REVIEW

**Comment by ReneWerner87** [comment]
hi thanks for the work

is this not already possible, just trying to understand what your customization is supposed to do

can you explain this in more detail and if necessary send an example, which shows what was not possible before and is now possible

**Comment by ReneWerner87** [comment]
think i now understand the point of this pull requests

nevertheless please describe briefly in the pull request with an example, what is possible now, what did not work before

so that we have this stored here for later

can you please also extend the test with such a case
in: https://github.com/gofiber/fiber/blob/master/app_test.go

File: app.go [hunk 1]
```diff
@@ -974,6 +975,11 @@ func (app *App) Test(req *http.Request, msTimeout ...int) (resp *http.Response,
 		return nil, err
 	}
 
+	// adding back the query from URL, since dump cleans it
+	dumps := bytes.Split(dump, []byte(" "))
```
Comment (ReneWerner87) [inline app.go:979]: please improve this construct with regard to performance and allocations

Comment (ancogamer) [reply app.go:979]: I will take a better a look on this. The problem is: imagine you have a query /foo?bar=boo, when the dump is create the query part is removed, staying /foo.

**Pull request merged**